### PR TITLE
Handle “busy signal” appropriately.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/call/Speakerbox/CallKitCallUIAdaptee.swift [hunk 1]
```diff
@@ -145,24 +155,35 @@ final class CallKitCallUIAdaptee: NSObject, CallUIAdaptee, CXProviderDelegate {
...
     func remoteDidHangupCall(_ call: SignalCall) {
         AssertIsOnMainThread()
+        Logger.debug("\(self.TAG) \(#function)")
 
         provider.reportCall(with: call.localId, endedAt: nil, reason: CXCallEndedReason.remoteEnded)
     }
 
+    func remoteBusy(_ call: SignalCall) {
+        AssertIsOnMainThread()
+        Logger.debug("\(self.TAG) \(#function)")
```
Comment (charlesmchen) [inline Signal/src/call/Speakerbox/CallKitCallUIAdaptee.swift]: Maybe I should take out this logging.

**Comment by charlesmchen** [comment]
PTAL @michaelkirk

**Comment by michaelkirk** [comment]
LGTM.

**Comment by charlesmchen** [comment]
Merged into master.

**Pull request closed without merging**